common/types.go

```diff
@@ -163,8 +164,77 @@ func (ep *EvictionPolicy) Parse(s string) error {
 
 type LogConfig struct {
 	Level       LogLevel
-	FilePath    string
 	MaxFileSize uint64
 	FileCount   uint64
+	FilePath    string
 	TimeTracker bool
 }
+
+type Block struct {
```

Comment:
There are not very common types. Only stream and azstorage components will need it. Shall we move them to a new file and name it accordingly which clearly says its related to streaming.

Reply:
I think we should keep it in common, it would be a little strange creating a file used explicitly by stream and azstorage